pmm-app/tests/verifyQAN_test.js

```diff
@@ -213,6 +213,8 @@ xScenario('PMM-T123 - Verify User is able to search for filter value', async (I,
     'generic',
   ];
   qanPage.waitForNewQANPageLoaded();
+  I.click(qanPage.fields.last5MinutesButton);
```

Comment:
we can just use the filter url for last 5 mins, and why do you need to do 12 hours also?